simulator/mips/mips_instr_decode.h

```diff
@@ -61,6 +70,18 @@ struct MIPSInstrDecoder
         }
     }
 
+    uint32 get_fmt( char type) const
+    {
+        switch ( type)
```

Comment:
Don't see how you're using that method...

Reply:
I was not quite sure what that 5th argument stood for so I thought it might be used as an indicator of what type of float number we are working with. I see that it is useless and I'll delete it.